src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationUpdateCommandTest.php

```diff
@@ -91,7 +91,6 @@ protected function setUp()
     {
         $this->fs = new Filesystem();
         $this->translationDir = sys_get_temp_dir().'/'.uniqid('sf_translation', true);
-        $this->fs->mkdir($this->translationDir.'/Resources/translations');
         $this->fs->mkdir($this->translationDir.'/Resources/views');
```

Comment:
We can remove this too, I guess.

Reply:
Yea, this PR needs work, will finish tomorrow :)